Source/Dnmh.Security.ApiKeyAuthentication.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Kunne der være en fordel i at bibeholde support for de ældre .NET versioner?
Det har jeg gjort i en del af de andre NuGet pakker vi har.
I hvert fald beholdt dem som ikke er forbi end-of-life, dvs. .NET 8 og .NET 9
```suggestion
    <TargetFrameworks>net9.0;net8.0;net7.0</TargetFrameworks>
```

---

Jeg har tilføjet .Net 8. 

Der er breaking changes fra .Net 7 til 8, som gør at jeg skulle indføre preprocessor directives der tjekker på .Net versionen og det synes jeg alligevel var for meget i forhold til at .Net 7 er EOL.